src/app/investment/investment.service.ts

```diff
@@ -151,7 +167,7 @@ for(let x=0; x<this.portfolio.length;x++){
 
 
    }
-   if (this.portfolio[z].type == 'Real Estate' || this.portfolio[z].type == 'realestate'){
+   if (this.portfolio[z].type == 'Real Estate' || this.portfolio[z].type == 'Urban Real Estate' || this.portfolio[z].type == 'Rural Real Estate' || this.portfolio[z].type == 'realestate'){
```

Comment:
Typescript/Javascript uses a triple equal operator (===). Double equal doesn't check for strict equality. So two values of different types would equal each other such as 23 would equal '23' even though one is an integer and the other is a string.